llvm/test/CodeGen/Mips/llvm-ir/sdiv.ll

```diff
@@ -206,12 +234,30 @@ entry:
 }
 
 define signext i32 @sdiv_i32(i32 signext %a, i32 signext %b) {
-; GP32-LABEL: sdiv_i32:
-; GP32:       # %bb.0: # %entry
-; GP32-NEXT:    div $zero, $4, $5
-; GP32-NEXT:    teq $5, $zero, 7
-; GP32-NEXT:    jr $ra
-; GP32-NEXT:    mflo $2
+; GP32R0R1-LABEL: sdiv_i32:
```

Comment:
I based the naming here off the entries that already existed for MIPS II, my changes are specifically for MIPS I, not Mips32R1, though I agree the naming scheme is very confusing.  Would you recommend changing the names for both mips1 and mips2?

Reply:
Since MIPS I is not supported well yet for LLVM, and I guess the 32R0 was supposed to be equal with MIPS II in this context.

If we need MIPS I, we may use a new name for it, maybe 32_I?